GLOSSARY.md

```diff
@@ -41,7 +41,7 @@ This explains high-level terms that have a specific meaning within Hyrise and th
 | Column Type                | Type of a class inheriting from BaseColumn (mostly used as template parameter name)                                                                                        |
```

Comment:
> I have not looked into all of the BP128 code - we should have a single entry point with a deeper explanation of the memory layout in the code, aybe as an ascii art.

There are images in the linked paper. This is very specialised code that will probably only be ever touched by a few people. Everyone else doesn’t need to completely understand it in order to use it. An ASCII image will be difficult, because one line would be at least 128 bits.